src/molecules/form-fields/password-form-field.tsx

```diff
@@ -50,7 +51,7 @@ const PasswordFormField: React.FC<PasswordFormFields> = (
     } = props;
 
     const [isVisible, setIsVisible] = useState<boolean>(false);
-    const cssIsValid = isValid ? "" : "-invalid";
+    const cssIsValid = isValid ? "" : InvalidInputClassName;
```

Comment:
Could this be namespaced to this component so we don't (potentially) have multiple `InvalidInputClassName` imports? `PasswordFormFieldInvalidClassName`

Reply:
Thanks for the feedback. I have changed the name if you want to take another look. 👍